boards/arm/nucleo_f103rb/nucleo_f103rb.dts

```diff
@@ -30,10 +30,37 @@
 		compatible = "gpio-keys";
 		user_button: button {
 			label = "User";
-			gpios = <&gpioc 13 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpioc 13 GPIO_ACTIVE_LOW>;
 		};
```

Comment:
Do not get why these two changes are in this commit and not in 717de74

Reply:
Indeed, I pushed these initially for testing purpose.
I'll likely reshuffle the commits once all tests are done.